### PR TITLE
Fix namespace validation in 1.9

### LINKED ISSUE #638: ASM 1.9 script is failing to validate when istio-system namespace doesn't exist.
In our ASM installation pipeline we have a step to validate installation, which looks like this:

```
${SCRIPTS_PATH}/${ASM_VERSION}/install_asm \
  --project_id $PROJECT_ID \
  --cluster_name $CLUSTER_NAME \
  --cluster_location $CLUSTER_LOCATION \
  --mode $ASM_MODE \
  --custom_overlay "${SCRIPTS_PATH}/${ASM_VERSION}/overlays/options/internal-load-balancer-${CLUSTER_NAME}.yaml" \
  --output_dir ${INSTALLATION_PACKAGE_PATH} \
  --only_validate
```

But this is reporting an error:
```
13:15:21  install_asm: Checking for istio-system namespace...
13:15:21  install_asm: [ERROR]: The istio-system namespace doesn't exist.
13:15:21  Please create the "istio-namespace" and retry, or run the script with the
13:15:21  '--enable_namespace_creation' flag to allow the script to enable it on your behalf.
13:15:21  Alternatively, use --enable_all|-e to allow this tool to handle all dependencies.
13:15:21  + cleanup
13:15:21  + rm -rf /tmp/kubectl_wrapper_5150_29403
```

Then when we try to execute script with this flag: --enable_namespace_creation, it is throwing another exception:
```
install_asm: [ERROR]: Unknown option --enable_namespace_creation
10:37:51  install_asm 1.9.3-asm.2+config2
10:37:51  usage: install_asm [OPTION]...
10:37:51  
10:37:51  Set up, validate, and install ASM in a Google Cloud environment.
10:37:51  Use -h|--help with -v|--verbose to show detailed descriptions.
10:37:51  
10:37:51  OPTIONS:
10:37:51    -l|--cluster_location  <LOCATION>
10:37:51    -n|--cluster_name      <NAME>
```

And we see that in the code there is no handler for that flag "enable_namespace_creation" at all?

For a short term we will create this NS manually before executing asm_install script, but this is not documented anywhere.

Also documentation is assuming the executor has an elevated permission and can run with flag --enable-all, which is not a case here.

Thanks

### PR DESCRIPTION
**Pull request by zerobfd**
Fixes #638

### CODE REVIEW

File: scripts/asm-installer/install_asm [hunk 1]
```diff
@@ -2229,14 +2229,13 @@ create_istio_namespace() {
...
+warn_if_istio_namespace_not_exists() {
   info "Checking for istio-system namespace..."
   if ! istio_namespace_exists; then
-    { read -r -d '' MSG; fatal "${MSG}"; } <<EOF || true
+    { read -r -d '' MSG; warn "${MSG}"; } <<EOF || true
 The istio-system namespace doesn't exist.
-Please create the "istio-namespace" and retry, or run the script with the
-'--enable_namespace_creation' flag to allow the script to enable it on your behalf.
-$(enable_common_message)
+Please create the "istio-namespace", or make sure that the user installing ASM
```
Comment (jasonwzm) [inline scripts/asm-installer/install_asm:2237]: What is "istio-namespace"?

Comment (zerobfd) [reply scripts/asm-installer/install_asm:2237]: Oops, thanks.

**Review by jasonwzm — APPROVED** [review]

File: scripts/asm-installer/install_asm (hunk 1 above)
Comment (shako92) [reply scripts/asm-installer/install_asm]: Yep I think there should be "istio-system"

**Comment by giulianobr** [comment]
Hey all, this bug is back in the 1.10 script.

**Pull request merged**